src/app/components/FetchCryptoData.tsx

```diff
@@ -1,29 +1,86 @@
 import React from 'react'
 
+import CryptoCard from './CryptoCard';
+
+
+
 const FetchCryptoData = () => {
     const [state, setState] = React.useState({
         data: null,
         error: null
     });
 
+    const [search, setSearch] = React.useState("");
+
     React.useEffect(() => {
-        fetch("https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd")
+        console.log(state.data)
+        fetch(
+          "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false"
```

Comment:
we could pull this API fetch logic into a hook and DRY up this logic for other uses.